docs/getting-started.md

```diff
@@ -123,7 +123,7 @@ In your **second shell** prompt:
       kernel="${image_bucket_url}/quickstart_guide/x86_64/kernels/vmlinux.bin"
```

Comment:
Could we check that the x86 example still works?

Reply:
Sure, I can check tomorrow.

In the meantime I found https://github.com/firecracker-microvm/firecracker/pull/2514 - the links were changed on purpose to make balloon work. Apparently I just changed it back - so we would need a working kernel with balloon for aarch64

Tagging @serban300 @RoyceDavison - can you help?